stacks/dovetail.prx.org.yml

```diff
@@ -124,6 +129,30 @@ Resources:
       DefaultActions:
         - Type: forward
           TargetGroupArn: !Ref DovetailApplicationALBTargetGroup
+  DovetailALBHTTPListenerRule:
+    Type: "AWS::ElasticLoadBalancingV2::ListenerRule"
+    Properties:
+      Actions:
+        - TargetGroupArn: !Ref DovetailApplicationALBTargetGroup
+          Type: forward
+      Conditions:
+        - Field: host-header
+          Values:
+            - dovetail.*
```

Comment:
IIRC, you get charged based on how many listener rules a request goes through.  So this sets up any `dovetail.prxu.org` requests as the first rule.  Then `dovetail3.*` requests.  Then the default (also the old Dovetail node app).